packages/moonstone/PictureInGraphics/PictureInGraphics.less

```diff
@@ -0,0 +1,56 @@
+@import '../styles/skin.less';
+@import '../styles/text.less';
+@import '../styles/mixins.less';
+
+@moon-pig-width:      321px;
+@moon-pig-border-width: 6px;
+@moon-pig-overlay-text-margin-bottom: 9px;
+
+.pictureInGraphics {
+	display: block;
+	position: relative;
+	padding: @moon-pig-border-width;
+	width: @moon-pig-width;
+	height: auto;
+
+	.video {
+		display: block;
+		width: 100%;
+		height: 100%;
+		margin: 0;
+	}
+
+	.image {
+		box-sizing: border-box;
+		display: block;
+		position: absolute;
```

Comment:
Whenever image placeholder is declared, the video won't be visible. Is this intentional? My guess is for locked/restricted channels may need that, but wanted to confirm. If so let's document it.